mriqc/bin/mriqc_run.py

```diff
@@ -184,6 +195,9 @@ def main():
         'webapi_url' : opts.webapi_url,
         'webapi_port' : opts.webapi_port,
         'upload_strict' : opts.upload_strict,
+        'T1w_file' : opts.T1w_file,
```

Comment:
yes for now I only test T1w_file, but the plane is to extend it to other modality
so not use now but may be ... soon